### DISCUSSION TITLE
Moderation

### DISCUSSION THREAD

**Discussion by glennawatson** [dotnet-foundation/Home#50] [General Discussions]
Some people have found the recent discussions toxic.

Should there be a moderation policy or similar in terms of handling the public discussion areas?

Discussions recently seemed to have a lot of feedback that hasn't been public domain for a while, how do you balance the fairness factor with discussions going forward so people don't feel penalised for speaking their mind as well.

**Comment by mitchelsellers** [comment]
I believe that we need to have an open communication channel, and it seems that GitHub is going to be the location for this.  However, I agree that some of the commentaries in the recent discussions have far exceeded the value of learning frustrations.

We are all bound by the[ .NET Foundation Code of Conduct](https://dotnetfoundation.org/about/code-of-conduct) as part of our project contributions, I think at a minimum those same rules should apply to the discussions in the policy. Which, in fact, is already stated anytime you post a reply.  So enforcing those policies that already exist is most likely a smart place to start.  Personal attacks, demanding people resign, etc go against the spirit of those rules for sure.

**Reply by eatdrinksleepcode** [reply]
You may disagree that resignation was warranted in this case, but asking for a resignation when a severe breach of trust occurs is not an unexpected or unusual occurrence in an organization like this one, and it is most certainly not against the rules.

**Reply by mitchelsellers** [reply]
Honestly this comment isn’t necessarily about her resignation. But more about the general tone and other commentary. 

We need to be respectful adults, ALWAYS. 

Egregious use of power does have consequences and you are correct that requesting resignation Ivan be warranted. But it’s the other comments and accusations that most likely violate the rules that exist for the organization already. 

If we have rules, the rules should be enforced. If not, the rules shouldn’t be there.

**Reply by mossyblog** [reply]
> You may disagree that resignation was warranted in this case, but asking for a resignation when a severe breach of trust occurs is not an unexpected or unusual occurrence in an organization like this one, and it is most certainly not against the rules.

You could mount a case it breaches the Code of Conduct, specifically:

- Demonstrating empathy and **_kindness toward other people_**
- Being respectful of differing opinions, **_viewpoints_**, and **_experiences_**
- Giving and gracefully **_accepting constructive feedback_**
- Accepting responsibility and apologizing to those affected by our **mistakes**, and **_learning_** from the experience

There was no real dialogue, it just escalated to "resign or else". 

As to who moderates the discussion and under what constraints - the Code of Conduct isn't explicit either which leaves a lot of room to interpretation and status of the respondents within the community. Not to mention maturity displayed by "contributors" isn't exactly putting the best foot forward here on ability to handle conflict negotiation.

**Reply by eatdrinksleepcode** [reply]
> You could mount a case it breaches the Code of Conduct, specifically:

No, you couldn't. Not one of the points that you listed prohibits someone from calling for a resignation if they believe it is warranted given the circumstances.

_You_ may not believe it is warranted, but that is your opinion. And attempting to suppress the opinion of others using the threat of a CoC violation as a bludgeon is exactly the kind of toxic behavior that makes people hesitant to use CoCs in the first place.

**Reply by gortok** [reply]
While I don’t agree that Claire should have resigned or that her resignation should have been called for; using the CoC to keep from holding power to account is the exact opposite of its purpose.

**Reply by abowen** [reply]
> No, you couldn't. Not one of the points that you listed prohibits someone from calling for a resignation if they believe it is warranted given the circumstances.

I agree it's a grey area but the issue was that there was hardly dialogue.

"If the new Board is serious about rectifying the situation at hand here I believe that several actions need to be taken within the next 24 hours." which includes her resignation.

Judge, jury and executioner.

The one person who says "How about we don't do a knee jerk reaction?"

https://github.com/dotnet-foundation/Home/discussions/39#discussioncomment-1444219

Then gets more thumbs down + told no, Claire was being evil. That's hardly demonstrating the CoC listed

**Reply by abowen** [reply]
> While I don’t agree that Claire should have resigned or that her resignation should have been called for; using the CoC to keep from holding power to account is the exact opposite of its purpose.

Except no one is intending that from what I read above.

It was more along the lines of "Instead of doing a knee jerk reaction, how about we let Claire or the board reply to these different issues we've raised?"

If after those replies, it didn't address the community concerns, then sure, I would feel more comfortable with people requesting her / the boards resignation

**Reply by JimBobSquarePants** [reply]
Hey @mossyblog this you?

https://github.com/dotnet-foundation/Home/discussions/39#discussioncomment-1452851

**Reply by glennawatson** [reply]
Worth noting there have been a few calls for resignations recently. Claire the one discussed above, mine and the boards.

**Reply by IanKemp** [reply]
The only toxicity I saw was from those implying that any criticism of Claire's actions and/or calls for her resignation is somehow unwarranted and/or only due to her being a woman. After she resigned, similar claims were made.

Yes, there is a misogyny problem in tech.
No, white knighting does not solve that problem; it makes it worse, because it's sexist in and of itself.

If you have evidence that Claire was mistreated due to her gender in any way, provide it. If you don't, stop making her into a martyr. She deserves better than to be judged for her actions, or excused for them, through the lens of gender. Based on the evidence I have, she is simply someone who made a mistake and took full responsibility for said mistake - were that our politicians could be so honest.

**Reply by abowen** [reply]
> The only toxicity I saw was from those implying that any criticism of Claire's actions and/or calls for her resignation is somehow unwarranted and/or only due to her being a woman. After she resigned, similar claims were made.

I do not see any of this in the GitHub discussions

https://github.com/dotnet-foundation/Home/discussions/39
https://github.com/dotnet-foundation/Home/discussions/40

Maybe I missed a new reply, I used the search terms "female", "woman", "sex", "gender"

No one above made reference to gender. Please don't add a strawman around white knighting to this discussion.
